pytealext/state.py

```diff
@@ -66,6 +72,10 @@ def put(self, value: Expr) -> App:
     def get(self) -> App:
         return App.localGet(Int(0), self._name)
 
+    def exists(self) -> Expr:
+        mb = App.localGetEx(Int(0), Int(0), self._name)
+        return Seq(mb, mb.hasValue())
+
```

Comment:
do you think it's worth adding test for that? e.g. 
```
If(mb.exists()).Then(Approve()).Else(Reject())
```

Additionally what does mb stands for?

Reply:
mb for maybe of the maybevalue

Reply:
changed mb to ex